pkg/kv/kvclient/kvcoord/txn_interceptor_committer_test.go

```diff
@@ -243,9 +243,7 @@ func TestTxnCommitterStripsInFlightWrites(t *testing.T) {
 	ba.Requests = nil
 	etArgsWithTrigger := etArgs
 	etArgsWithTrigger.InternalCommitTrigger = &roachpb.InternalCommitTrigger{
-		ModifiedSpanTrigger: &roachpb.ModifiedSpanTrigger{
-			SystemConfigSpan: true,
-		},
+		ModifiedSpanTrigger: &roachpb.ModifiedSpanTrigger{},
```

Comment:
Do we need to to set this to anything at all?

Reply:
Looks like just `InternalCommitTrigger` is enough

Reply:
Nope